config/default.yml

```diff
@@ -3962,6 +4159,29 @@ Style/InverseMethods:
     :select: :reject
     :select!: :reject!
 
+Style/InvertibleUnlessCondition:
+  Description: 'Favor `if` with inverted condition over `unless`.'
+  Enabled: false
```

Comment:
Why disabling this? Sounds reasonable to us.

Reply:
`Enabled: false` is Rubocop's default. I did not change any of those.

Reply:
Thinking about it, this cop might be too biased. Not sure if these are must-haves:
```
unless number.zero? => if number.nonzero?
unless items.any? => if items.none?
```
Leaving it disabled for now.